### PR TITLE
fix(color) - Adjust page padding for pre-flight checks, trims and throttle setup pages.

### LINKED ISSUE #2534: Padding is missing on some model setting pages
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### What part of EdgeTX is the focus of this bug?

Transmitter firmware

### Current Behavior

The paddings are missing on some pages which open from the Model Setup pagetab (Color UI). So the texts are too left and up. These are the affected pages:
- Preflight Checks
- Trims
- Throttle

### Expected Behavior

They should be the same like the RF setting and Timer pages.

### Steps To Reproduce

Just open them from the Model Setup page.

### Version

Nightly (Please give date/commit below)

### Transmitter

Radiomaster TX16S / TX16SMK2

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by philmoz**
Fixes #2534 

Also made the button size consistent and display better pot/slider name in the center beeps buttons in the pre-flight checks page.

Screen Shot 2023-03-03 at 8 11 55 am

Screen Shot 2023-03-03 at 8 06 38 am

### CODE REVIEW

**Comment by Eldenroot** [comment]
Great, looks much better!

Also would be good to consider a layout for this https://github.com/EdgeTX/edgetx/issues/3110 , just think about it, because it will require more changes in gui for this page.

**Comment by philmoz** [comment]
Add padding to radio setup pages (Sound, Vario, Haptic, Alarm, Backlight and GPS).
Move backlight inactivity timer to its own line and add a label (requires translation).

Screen Shot 2023-03-04 at 12 23 05 pm

**Comment by ulfhedlund** [comment]
@philmoz: In all the ongoing improvements in the GUI I've noticed that a few of them requires translation.
If possible, it think it would be great if the translators then could be notified. See for example PR #3281 for a list of translators.

**Comment by philmoz** [comment]
> @philmoz: In all the ongoing improvements in the GUI I've noticed that a few of them requires translation.
> If possible, it think it would be great if the translators then could be notified. See for example PR #3281 for a list of translators.

In this case it might be best to wait until the PR is approved in case anyone wants to change my initial choice for the new 'Inactivity timeout' label.

**Comment by Eldenroot** [comment]
I like it!

File: radio/src/translations/cz.h [hunk 1]
```diff
@@ -743,6 +743,7 @@
 #define TR_DELETE_THEME                "Smazat motiv?"
 #define TR_SAVE_THEME                  "Uložit motiv?"
 #define TR_EDIT_COLOR                  "Edit Color"
```
Comment (Eldenroot) [inline radio/src/translations/cz.h:745]: Upravit barvu

File: radio/src/translations/cz.h [hunk 2]
```diff
@@ -743,6 +743,7 @@
...
+#define TR_BACKLIGHT_TIMER             "Inactivity timeout"
```
Comment (Eldenroot) [inline radio/src/translations/cz.h:746]: Doba pro zhasnutí

**Review by Eldenroot — CHANGES_REQUESTED** [review]

File: radio/src/translations/cz.h (hunk 1 above)
Comment (philmoz) [reply radio/src/translations/cz.h:745]: This is different to the translation you posted for #3278  (Změnit barvu)

**Comment by pfeerick** [comment]
The initial translation is fine by me, but the pause to allow for review is appreciated. 

Perhaps for a further tweak to the backlight page, is it worth making it so that when you select 'ON' as the mode that the 'inactivity timeout' and 'off brightness' controls/labels are hidden (or if a page rebuild is feasible, completely hidden) since they are no longer relevant (only valid for the other modes).

**Comment by pfeerick** [comment]
Can we get some translations for this please 😃 

```
#define TR_BACKLIGHT_TIMER "Inactivity timeout"
```

image


- [ ] @zyren CN / TW
- [ ] @Eldenroot CZ
- [ ] @HThuren DA
- [ ] @ParkerEde / @TheIsotopes DE
- [ ] @robustini IT
- [ ] @ToshihiroMakuuchi JP
- [ ] @ajjjjjjjj PL
- [ ] @ulfhedlund SE

**Comment by philmoz** [comment]
> Perhaps for a further tweak to the backlight page, is it worth making it so that when you select 'ON' as the mode that the 'inactivity timeout' and 'off brightness' controls/labels are hidden (or if a page rebuild is feasible, completely hidden) since they are no longer relevant (only valid for the other modes).

The controls were disabled; but hiding them makes more sense - added now.

**Comment by pfeerick** [comment]
derp... so they were... just there was no visible sign of it (dimmed/greyed out, etc)... so this is perfect...

**Comment by robustini** [comment]
IT

#define TR_BACKLIGHT_TIMER "Tempo d'inattività"

**Comment by HThuren** [comment]
DA
#define TR_BACKLIGHT_TIMER TR2("Inaktivitet", "Ved inaktivitet dæmp efter")

**Comment by ulfhedlund** [comment]
SE

#define TR_BACKLIGHT_TIMER "Vid inaktivitet släck efter"

**Comment by TheIsotopes** [comment]
DE

`#define TR_BACKLIGHT_TIMER             "Inaktivitäts Timeout"`

**Comment by zyren** [comment]
**********************************
CN

#define TR_BACKLIGHT_TIMER             "持续时间"
***********************************
**********************************
TW

#define TR_BACKLIGHT_TIMER             "持續時間"
***********************************

**Comment by ToshihiroMakuuchi** [comment]
JP
#define TR_BACKLIGHT_TIMER "持続時間"

File: radio/src/translations/cz.h (hunk 1 above)
Comment (Eldenroot) [reply radio/src/translations/cz.h:745]: Both are right... Czech language has many synonyms :) 

Please leave "Upravit barvu"

**Review by Eldenroot — CHANGES_REQUESTED** [review]

**Review by Eldenroot — CHANGES_REQUESTED** [review]

**Comment by ajjjjjjjj** [comment]
PL
`#define TR_BACKLIGHT_TIMER "Czas bezczynności"`

**Pull request merged**